faq/tests/test_views.py

```diff
@@ -23,7 +23,7 @@ def setUpClass(cls):
     def test_pagination(self):
         response = self.client.get(self.faq.get_absolute_url())
 
-    @ classmethod
+    @classmethod
     def tearDownClass(cls):
         del cls.client
```

Comment:
You're missing a call to `super`. Also, this is not required at all. Change from setUpClass to setUpTestData. And inherit the base test class from django.test, not unittest. You don't have to explicitly define `client` then. Sorry, I know it's a bit of work but it is the latest Django way.